.agents/workflows/coding-standards.md

```
@@ -209,26 +236,270 @@ export type InsertMatter = typeof matters.$inferInsert;
+❌ **DON'T:**
+- Return `Result<T>` objects
+- Throw custom error classes
+- Log and throw (let handler log)
+- Omit context from errors
+- Mix old HTTPException with new factory functions
+
+## 7b. Error Flow Architecture
+
+Errors flow through a single pipeline — handlers never catch service errors:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)